### PR TITLE
[stable-4.5] Add UIClient class

### PR DESCRIPTION
**Pull request by jerabekjiri**
No-Issue

Fix `stable-4.5` CI issues and set `continue-on-error: false`

Issues:
- UIClient was missing (Started happening from https://github.com/ansible/galaxy_ng/pull/1604)
-  Skip `test_galaxy_openapi_validation` (https://github.com/ansible/galaxy_ng/pull/1616)

### CODE REVIEW

**Comment by rochacbruno** [comment]
need to change this line https://github.com/ansible/galaxy_ng/actions/runs/5015866219/workflow?pr=1726#L58 so Ci will report the error

**Review by chr-stian — CHANGES_REQUESTED** [review]
pls take a look at this
https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/tests/integration/utils/client_ui.py#L49-L69
we need self._rs.verify = False and the referer, also in post and delete methods

File: galaxy_ng/tests/integration/cli/test_dependencies.py [hunk 1]
```diff
@@ -68,11 +70,17 @@ def test_collection_dependency_install(ansible_config, published, cleanup_collec
...
+    # wait for move task from `inbound-<namespace>` repo to `staging` repo
+    time.sleep(SLEEP_SECONDS_ONETIME)
+
     if retcode == 0:
         config = ansible_config("ansible_insights")
         client = get_client(config)
         set_certification(client, artifact2)
 
+        # wait for move task from `staging` repo to `published` repo
+        time.sleep(SLEEP_SECONDS_ONETIME)
```
Comment (chr-stian) [inline galaxy_ng/tests/integration/cli/test_dependencies.py:82]: we have the method `wait_for_url`

`    dest_url = (
        f"v3/plugin/ansible/content/{level}/collections/index/"
        f"{collection.namespace}/{collection.name}/versions/{collection.version}/"
    )
    wait_for_url(client, dest_url)`

Comment (jerabekjiri) [reply galaxy_ng/tests/integration/cli/test_dependencies.py:82]: we don't have `wait_for_url` in `stable-4.5`, it was added in 4.6, but I can try to add it to PR

**Pull request merged**